src/Helper/Utils.php

```diff
@@ -1,66 +1,14 @@
 <?php
+/**
+ * Helper Utilities
+ *
+ * @package Courier\Helper
+ */
 
 namespace Courier\Helper;
 
 /**
- * Class Utils
- * @package LinchpinHelpdesk\Helper
+ * Utils Class
  */
 class Utils {
-
-	/**
-	 * @param     $second
-	 * @param int $precision
-	 *
-	 * @throws \Exception
-	 */
-	public static function round_time( $second, $precision = 30 ) {
```

Comment:
@nate-allen were these methods not used?

Reply:
Went through. I am guessing that these methods weren't used anymore so that's why they were removed. But that's the only question on had from the review